cpp/src/arrow/flight/sql/odbc/tests/connection_test.cc

```diff
@@ -978,7 +962,7 @@ TEST(SQLGetDiagRec, TestSQLGetDiagRecForConnectFailure) {
 
   EXPECT_TRUE(ret == SQL_SUCCESS);
 
-  EXPECT_GT(message_length, 200);
```

Comment:
The error message from mock server is only 129 characters, so I adjusted this check